src/vs/workbench/contrib/chat/test/common/__snapshots__/Response_inline_reference.0.snap

```diff
@@ -9,15 +9,7 @@
     kind: "markdownContent"
   },
   {
-    inlineReference: {
-      scheme: "https",
-      authority: "microsoft.com",
-      path: "/",
-      query: "",
-      fragment: "",
-      _formatted: null,
-      _fsPath: null
-    },
+    inlineReference: URI(https://microsoft.com/),
```

Comment:
Why is this? I though the symbols don't make it into JSON stringified content

Reply:
Already created an issue: https://github.com/microsoft/vscode/issues/220048
The debug description symbol is used in `formatValue` to compute the snapshot value...